pombola/south_africa/templates/info/_blog_sidebar.html

```diff
@@ -14,7 +14,7 @@
       {% endfor %}
     </ul>
 
-    {% if popular_posts.exists %}
+    {% if some_popular_posts %}
```

Comment:
Or I guess just the date bit could be in the first query if that doesn't slow it down.